assets/wizards/performance/views/intro/index.js

```diff
@@ -0,0 +1,64 @@
+/**
+ * Performance Wizard Intro screen
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { Component, Fragment } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { Card, withWizardScreen } from '../../../../components/src';
+
+/**
+ * Intro screen for Performnance Wizard
+ */
+class Intro extends Component {
+	/**
+	 * Render.
+	 */
+	render() {
+		const { pluginRequirements } = this.props;
+		return (
+			<Fragment>
+				<Card>
+					<p>
+						{ __(
+							'Newspack utilizes PWA to automatically optomizing and configuring your news site to perform better. It improves:'
```

Comment:
```suggestion
							'Newspack utilizes PWA for automatically optimizing and configuring your news site to perform better. It improves:'
```

Reply:
I believe the many changes in https://github.com/Automattic/newspack-plugin/pull/141/commits/00104767f52267105b092ea7058b38384bfca4b0 ended up replacing this text anyway.